### ISSUE TITLE
Create page on how to encrypt storage devices in Windows/Mac

### ISSUE THREAD

**Issue by Zaxim** [MrVaughan/Security-Best-Practices#18]
We should have a page on how to use bitlocker and FileVault2.

I also would love to recommend a cross-platform solution, but Truecrypt is the only one that ever worked well, and it's been "deprecated." Are there other ones out there?

**Comment by MrVaughan** [comment]
Agreed that a cross platform solution would be ideal. I'm not an expert in this area (especially the windows stuff). Mostly I use Full disk encryption with nix, and haven't don't much with the other stuff other then encrypted volumes with truecrypt.
